src/coreclr/jit/lowerarmarch.cpp

```diff
@@ -74,6 +74,7 @@ bool Lowering::IsContainableImmed(GenTree* parentNode, GenTree* childNode) const
             case GT_ADD:
             case GT_SUB:
 #ifdef TARGET_ARM64
+                return emitter::emitIns_valid_imm_for_add(immVal, size);
```

Comment:
This fixes an unrelated but important issue:
```csharp
static int Test(int x) => x + 1;
```
on any >=armv8.1 hardware we used to return false for GT_ADD in IsContainableImmed, it led to:
```
        00000000          mov     w1, #1
        00000000          add     w0, w0, w1
```
instead of:
```
        00000000          add     w0, w0, #1
```